apps/src/studio/paramLists.js

```diff
@@ -22,8 +31,29 @@ exports.getPlaySoundValues = function (withRandom) {
   return names;
 };
 
+/**
+ * Returns a list of sounds for our droplet playSound block.
+ */
+
 exports.playSoundDropdown = function () {
+  var skinSoundMetadata = utils.valueOr(skin.soundMetadata, []);
+
   return exports.getPlaySoundValues(true).map(function (sound) {
-    return '"' + sound + '"';
+    var lowercaseSound = sound.toLowerCase().trim();
+    var handleChooseClick = function (callback) {
+      var playbackOptions = $.extend({
+        volume: 1.0
+      }, _.find(skinSoundMetadata, function (metadata) {
+        return metadata.name.toLowerCase().trim() === lowercaseSound;
+      }));
+
+      studioApp.playAudio(lowercaseSound, playbackOptions);
+      callback(quote(sound));
+    };
+    return {
+      text: quote(sound),
+      display: quote(sound),
+      click: handleChooseClick
```

Comment:
yes, fortunately, it did! Applab uses it for one menu item in the assert dropdown where you can choose to add a new image.